packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -85,6 +85,24 @@ export const orchestratorApi = createApi({
                 const graphqlFn = graphqlRequestBaseQuery({
                     url: customApi ? customApi.apiBaseUrl : graphqlEndpointCore,
                     prepareHeaders,
+                    customErrors: (error) => {
+                        const { name, message, stack, response } = error;
+                        if (
+                            response &&
+                            response.errors &&
+                            response.errors.length > 0
+                        ) {
+                            response.errors.map((error) => {
+                                if (
+                                    error.extensions?.error_type ===
+                                    'not_authorized'
```

Comment:
I would suggest to change this to `not_authenticated` once it is available https://github.com/workfloworchestrator/orchestrator-core/issues/649

Reply:
I created frontend ticket and added it as a todo https://github.com/workfloworchestrator/orchestrator-ui-library/issues/1105